pages/contributors.tsx

```diff
@@ -73,83 +76,286 @@ const ContributorsPage: FC<{ contributors: Contributor[] }> = ({
     (a, b) => b.contributions - a.contributions
   )
 
-  const buttonStyles =
-    'bg-violet-600 hover:bg-transparent text-white px-4 py-2 md:px-3 text-sm tracking-[.6px] rounded-md border border-dashed border-transparent duration-100 hover:border-violet-400 hover:text-violet-500 dark:hover:text-violet-400'
-
   const linkProps = {
     target: '_blank',
     rel: 'noopener noreferrer',
-    className: buttonStyles,
   }
 
-  const isDarkMode = resolvedTheme === 'dark'
+  const iconsComponents: { [key: string]: JSX.Element } = {
+    GitHub: <FaGithub />,
+    Twitter: <FaXTwitter />,
+    Sponsor: <FaHeart />,
+    LinkedIn: <FaLinkedin />,
+    Website: <FaStaylinked />,
+  }
+
+  const maintainersLogins = maintainersData.map(
+    (maintainer) => maintainer.login
+  )
+
+  const contributorsWithoutMaintainers = sortedContributors.filter(
+    (contributor) => !maintainersLogins.includes(contributor.login)
+  )
 
-  const imageInfo = `image-effect w-9 h-9 rounded-full bg-gray-100 border text-lg text-gray-900 pl-[9px] pt-1 ${
-    isDarkMode ? '' : 'border-dashed border-violet-400'
-  } `
+  const getDarkBgColor = (id: number, check?: string) => {
+    if (id === 0) {
+      if (check === 'bg') {
+        return 'bg-[#EF6086] bg-opacity-5 dark:bg-opacity-10 dark:bg-[#EF6086]'
+      } else if (check === 'text') {
+        return 'text-[#EF6086]'
+      } else if (check === 'role') {
+        return 'bg-[#EF60861A] bg-opacity-10 dark:bg-[#EF6086] dark:bg-opacity-20 border dark:border-none border-[#EF6086] border-opacity-50'
+      } else if (check === 'hover') {
+        return 'hover:bg-[#EF6086] hover:bg-opacity-5 dark:hover:bg-[#EF6086] dark:hover:bg-opacity-20 hover:text-[#EF6086]'
+      } else if (check === 'border') {
+        return 'dark:border-[#EF6086] dark:border-opacity-20 border-[#EF6086]'
+      }
+    } else if (id === 1) {
+      if (check === 'bg') {
+        return 'bg-[#565990] bg-opacity-10 dark:bg-[#565990] dark:bg-opacity-30' // 565990
+      } else if (check === 'text') {
+        return 'text-[#565990] dark:text-[#1b1d49]'
+      } else if (check === 'role') {
+        return 'bg-[#565990] bg-opacity-10 dark:bg-[#8b8fca] dark:bg-opacity-90 border dark:border-none border-[#8b8fca] border-opacity-50'
+      } else if (check === 'hover') {
+        return 'hover:bg-[#565990] hover:bg-opacity-5 dark:hover:bg-[#565990] dark:hover:bg-opacity-20 hover:text-[#3f426d] dark:hover:text-[#8b8fca]'
+      } else if (check === 'border') {
+        return 'dark:border-[#565990] dark:border-opacity-20 border-opacity-50 border-[#565990]'
+      }
+    } else if (id === 2) {
+      if (check === 'bg') {
+        return 'bg-[#20BA88] bg-opacity-5 dark:bg-[#20BA88] dark:bg-opacity-10'
```

Comment:
I think `if else` statement looks weird here. Can you please try to use the object method instead?